### PR TITLE
Add apache-centos7-atomicapp

### PR DESCRIPTION
**Pull request by cdrage**

### CODE REVIEW

**Comment by dustymabe** [comment]
LGTM

**Comment by cdrage** [comment]
Changed the code a bit ^^.

So basically, we are just copying over helloapache to apache-centos7-atomicapp for compatibility reasons (docker build on hub.docker.com). 

In the future, I think we should rename helloapache to helloworld-atomicapp :)

File: apache-centos7-atomicapp/Nulecule [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+specversion: 0.0.2
+id: apache-centos7-atomicapp
+
+metadata:
+  name: Hello Apache App
+  appversion: 0.0.1
+  description: Atomic app for deploying a really basic Apache HTTP server
+graph:
+  - name: helloapache-app
```
Comment (dustymabe) [inline apache-centos7-atomicapp/Nulecule]: I wouldn't use "hello apache" in here any more

File: apache-centos7-atomicapp/Nulecule [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+specversion: 0.0.2
+id: apache-centos7-atomicapp
+
+metadata:
+  name: Hello Apache App
```
Comment (dustymabe) [inline apache-centos7-atomicapp/Nulecule]: I wouldn't use "hello apache" in here any more

**Comment by cdrage** [comment]
updated ^^

File: apache-centos7-atomicapp/Nulecule [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+specversion: 0.0.2
+id: apache-centos7-atomicapp
+
+metadata:
+  name: Hello Apache Atomic App
```
Comment (dustymabe) [inline apache-centos7-atomicapp/Nulecule]: hello apache

File: apache-centos7-atomicapp/Nulecule [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  appversion: 0.0.1
+  description: Atomic app for deploying a really basic Apache HTTP server
+graph:
+  - name: helloapache-atomicapp
```
Comment (dustymabe) [inline apache-centos7-atomicapp/Nulecule]: hello apache

File: apache-centos7-atomicapp/artifacts/kubernetes/hello-apache-pod.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
+    "apiVersion": "v1",
+    "kind": "Pod",
+    "metadata": {
+        "labels": {
+            "app": "helloapache"
```
Comment (dustymabe) [inline apache-centos7-atomicapp/artifacts/kubernetes/hello-apache-pod.json]: hello apache

File: apache-centos7-atomicapp/artifacts/kubernetes/hello-apache-pod.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        },
+        "name": "helloapache"
```
Comment (dustymabe) [inline apache-centos7-atomicapp/artifacts/kubernetes/hello-apache-pod.json]: hello apache

File: apache-centos7-atomicapp/artifacts/kubernetes/hello-apache-pod.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    },
+    "spec": {
+        "containers": [
+            {
+                "image": "$image",
+                "name": "helloapache",
```
Comment (dustymabe) [inline apache-centos7-atomicapp/artifacts/kubernetes/hello-apache-pod.json]: hello apache

File: apache-centos7-atomicapp/artifacts/marathon/helloapache.json [hunk 8]
```diff
@@ -0,0 +1,20 @@
+{
+  "id": "helloapache",
```
Comment (dustymabe) [inline apache-centos7-atomicapp/artifacts/marathon/helloapache.json]: hello apache

**Comment by cdrage** [comment]
Blah. Worst copy and paste ever by me.

Okay, updated and good-to-go @dustymabe

**Comment by dustymabe** [comment]
LGTM

**Pull request merged**